examples/website/i3s/webpack-debug.config.js

```diff
@@ -20,7 +20,15 @@ const CONFIG = {
         options: {
           presets: ['@babel/preset-env', '@babel/preset-react']
         }
-      }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/i,
```

Comment:
Can we use icons from https://fontawesome.com/ as soon as we already use this library?